### ISSUE TITLE
UnicodeEncodeError when launching label-studio

### ISSUE THREAD

**Issue by Rob192** [heartexlabs/label-studio#1068]
**Describe the bug**
When I am launching label-studio I am getting this error :
```python
UnicodeEncodeError: 'ascii' codec can't encode characters in position 4-47: ordinal not in range(128)
```

**To Reproduce**
Install label-studio via pip or via github

**Expected behavior**
Should be launching fine

**Screenshots**
This is the error log with the github install : 
```
=> Database and media directory: /root/.local/share/label-studio
=> Static URL is set to: /static/

[2021-06-16 14:50:16,437] [asyncio::__init__::54] [DEBUG] Using selector: EpollSelector
[2021-06-16 14:50:16,585] [urllib3.connectionpool::_new_conn::975] [DEBUG] Starting new HTTPS connection (1): pypi.org:443
[2021-06-16 14:50:21,780] [urllib3.connectionpool::_make_request::461] [DEBUG] https://pypi.org:443 "GET /pypi/label-studio/json HTTP/1.1" 200 35770
Watching for file changes with StatReloader
[2021-06-16 14:50:21,820] [django.utils.autoreload::run_with_reloader::617] [INFO] Watching for file changes with StatReloader
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/commands/runserver.py", line 110, in inner_run
    autoreload.raise_last_exception()
  File "/usr/local/lib/python3.6/dist-packages/django/utils/autoreload.py", line 76, in raise_last_exception
    raise _exception[1]
  File "/usr/local/lib/python3.6/dist-packages/django/core/management/__init__.py", line 357, in execute
    autoreload.check_errors(django.setup)()
  File "/usr/local/lib/python3.6/dist-packages/django/utils/autoreload.py", line 53, in wrapper
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.6/dist-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.6/dist-packages/django/apps/config.py", line 211, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/root/label-studio/label_studio/users/models.py", line 15, in <module>
    from organizations.models import OrganizationMember, Organization
  File "/root/label-studio/label_studio/organizations/models.py", line 11, in <module>
    from core.utils.common import create_hash, get_object_with_check_and_log, get_organization_from_request, load_func
  File "/root/label-studio/label_studio/core/utils/common.py", line 424, in <module>
    check_for_the_latest_version(print_message=True)
  File "/root/label-studio/label_studio/core/utils/common.py", line 416, in check_for_the_latest_version
    print(update_package_message())
UnicodeEncodeError: 'ascii' codec can't encode characters in position 4-47: ordinal not in range(128)
```

**Environment (please complete the following information):**
 - OS: Linux
 - Label Studio Version [latest from github]

**Comment by makseq** [comment]
@Rob192 
Could you try to do this command:

```
export PYTHONIOENCODING=utf8
```

More info is here: https://github.com/bitbrute/evillimiter/issues/41#issuecomment-543581149

**Comment by makseq** [comment]
@Rob192 I hope, you fixed your issue. Feel free to reopen it if this problem exists.

**Comment by Rob192** [comment]
Hello, 
Thank you very much for your answer ! It did work indeed but however, the font displayed inside the app were not so nice anymore (looking like Times New Roman). I think the issue arised because we annotated in french ... 
In between and for other users that might have the issue, I did manage to copy the folder locally and launch it with docker with mounting the copied folder. For some reason it did work (I could not use a docker on my server).
